spec.bs

```diff
@@ -3426,27 +3432,23 @@ of the following global objects:
           |generatedBidIDL|, |ig|, |expectedCurrency|, |isComponentAuction|, and |global|'s
           [=InterestGroupBiddingScriptRunnerGlobalScope/group has ad components=].
         1. If |bidOutput| is not failure, then set |generatedBid| to |bidOutput|.
+    1. Let |bidDebugReportInfo| be a new [=bid debug reporting info=].
+    1. Set |bidDebugReportInfo|'s [=bid debug reporting info/interest group owner=] to
+       |ig|'s [=interest group/owner=]
     1. Let |debugLossReportUrl| be |global|'s
       [=InterestGroupBiddingAndScoringScriptRunnerGlobalScope/debug loss report url=] if it's not
       failure, null otherwise.
+    1. Set |bidDebugReportInfo|'s [=bid debug reporting info/bidder debug loss report url=] to |debugLossReportUrl|.

```

Comment:
nit: since |debugLossReportUrl| is only used once here, we can inline it instead of declaring a variable using a step. (we were recommended to reduce steps and nests when it's trivial, since our spec algorithms are getting larger).

Reply:
I worry that the "if it's a failure, null otherwise"  stuff might be too hard to read if the two are combined?

Reply:
sure, readable is more important.